commitizen/cli.py

```diff
@@ -49,6 +49,11 @@
                         "action": "store_true",
                         "help": "show output to stdout, no commit, no modified files",
                     },
+                    {
+                        "name": "--signoff",
```

Comment:
I'd suggest also supporting git's interface directly `-s`:

```suggestion
                        "name": ["s", "--signoff"],
```

Reply:
ok, let me add this to the commit, and the docs
thansk

Reply:
I committed another version, I want to solve today:) thanks